powerauth-push-server/src/main/java/io/getlime/push/controller/rest/AdministrationController.java

```diff
@@ -236,13 +333,30 @@ public Response removeAndroid(@RequestBody ObjectRequest<RemoveAndroidRequest> r
      * @throws PushServerException Thrown when application credentials entity could not be found or request validation fails.
      */
     @RequestMapping(value = "huawei/update", method = { RequestMethod.POST, RequestMethod.PUT })
+    @Operation(summary = "Update Huawei configuration (deprecated)", description = "Update Huawei configuration endpoint (deprecated), use the /admin/app/hms/update endpoint")
     public Response updateHuawei(@Valid @RequestBody ObjectRequest<UpdateHuaweiRequest> request) throws PushServerException {
         final UpdateHuaweiRequest requestObject = request.getRequestObject();
-        logger.info("Received update Huawei request, application ID: {}", requestObject.getAppId());
+        logger.info("Received updateHuawei request, application ID: {}", requestObject.getAppId());
```

Comment:
We can apply logging guide i.e.
        logger.info("action: {}, state: {}, applicationId: {}", "updateHuawei", "initiated", requestObject.getAppId());
and in analogy to the whole controller

Reply:
Ok, I will change the logging in all classes.